client/app/app.js

```diff
@@ -1,79 +1,133 @@
-  "angularMoment",
-  "ovh-angular-toaster",
-  "ovh-angular-swimming-poll",
-  "oui",
+    "ovh-angular-sso-auth",
+    "ovh-angular-sso-auth-modal-plugin",
+    "ovh-api-services",
+    "ovh-common-style",
+    "ovh-angular-checkbox-table",
+    "ovh-angular-form-flat",
+    "ovh-angular-q-allSettled",
+    "ovh-angular-stop-event",
+    "ovh-angular-a-disabled",
+    "angularMoment",
+    "ovh-angular-toaster",
+    "ovh-angular-swimming-poll",
+    "oui",
+    "oui.list-view",
+    "chart.js",
 
-  "ovh-angular-pagination-front",
-  "ovh-angular-responsive-tabs",
-  "mgcrea.ngStrap.popover",
-  "mgcrea.ngStrap.tooltip",
-  "mgcrea.ngStrap.helpers.dimensions",
-  "mgcrea.ngStrap.core",
-  "ovh-angular-responsive-page-switcher",
-  "ovh-angular-responsive-popover",
-  "ovh-angular-sidebar-menu",
+    "ovh-angular-pagination-front",
+    "ovh-angular-responsive-tabs",
+    "mgcrea.ngStrap.popover",
+    "mgcrea.ngStrap.tooltip",
+    "mgcrea.ngStrap.helpers.dimensions",
+    "mgcrea.ngStrap.core",
+    "ovh-angular-responsive-page-switcher",
+    "ovh-angular-responsive-popover",
+    "ovh-angular-sidebar-menu",
 
-  "ng-slide-down",
-  "ovh-angular-jsplumb",
-  "tmh.dynamicLocale",
-  "ovh-angular-module-status",
-  "ovh-angular-otrs",
-  "ovh-api-services",
+    "ng-slide-down",
+    "ovh-angular-jsplumb",
+    "tmh.dynamicLocale",
+    "ovh-angular-module-status",
+    "ovh-angular-otrs",
+    "ovh-api-services",
 
-  "ovh-jquery-ui-draggable-ng",
-  "ovh-angular-jquery-ui-droppable",
-  "ovh-angular-slider",
-  "ng-at-internet",
-  "atInternetUiRouterPlugin",
-  "ngFlash",
-  "matchmedia-ng",
-  "ovh-angular-user-pref",
-  "ovh-angular-doc-url",
-  "ovhBrowserAlert"
+    "ovh-jquery-ui-draggable-ng",
+    "ovh-angular-jquery-ui-droppable",
+    "ovh-angular-slider",
+    "ng-at-internet",
+    "atInternetUiRouterPlugin",
+    "ngFlash",
+    "matchmedia-ng",
+    "ovh-angular-user-pref",
+    "ovh-angular-doc-url",
+    "ovhBrowserAlert"
 ])
+    .config(function ($stateProvider, TranslateDecoratorServiceProvider, TranslateServiceProvider) {
+        "use strict";
 
-.config(function ($stateProvider, TranslateDecoratorServiceProvider, TranslateServiceProvider) {
-    "use strict";
+        // Config current locale
+        TranslateServiceProvider.setUserLocale();
 
-    // Config current locale
-    TranslateServiceProvider.setUserLocale();
+        // Add translations decorator (need to be added before routes definitions)
+        TranslateDecoratorServiceProvider.add($stateProvider);
+    })
+    .config(function ($urlRouterProvider, $locationProvider) {
+        "use strict";
+        $urlRouterProvider.otherwise("/");
+        $locationProvider.html5Mode(false);
+    })
+    .config(function (responsivePopoverProvider) {
+        // tell to the module that we consider a mobile device with at least 800px width
+        responsivePopoverProvider.setMobileMediaQuery("(max-width: 800px)");
+    })
+    .config(ChartJsProvider => {
```

Comment:
Lines 77, 78 and 79 should not be here. It's for ChartJS and I think it not even used in IPLB branch...